### DISCUSSION TITLE
[Integration] Meta-level issues in Offloading: Scoping, Composition, Development, and their Automation

### DISCUSSION THREAD

**Discussion by niksu** [cucapra/latte21#19] [Papers]
Authors: André DeHon, Hans Giesen, Nik Sultana, Yuanlong Xiao
Paper PDF: https://capra.cs.cornell.edu/latte21/paper/11.pdf
Talk Recording: https://flightplan.cis.upenn.edu/latte21_video

**Comment by rachitnigam** [comment]
@niksu Can you upload your talk to YouTube? We'll be creating playlists for each session and need to have all the talks on YouTube.
